### PR TITLE
Release v0.25.0

### PR DESCRIPTION
**Pull request by plafer**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+  instead of the store's ([#274](https://github.com/cosmos/ibc-rs/issues/274))
+
+### IMPROVEMENTS
+
+- Remove `flex-error` and remove unused error variants([#164](https://github.com/cosmos/ibc-rs/issues/164))
+- ConnectionMsg::ConnectionOpen{Try, Ack} should not wrap a Box
+  ([#258](https://github.com/cosmos/ibc-rs/issues/258))
+- Track code coverage with `cargo-llvm-cov`
+  ([#277](https://github.com/cosmos/ibc-rs/issues/277))
+
```
Comment (hu55a1n1) [inline CHANGELOG.md:47]: We're missing an entry for v0.24.1-pre.1. Do we want to add one? Should we `yank` it or let it be?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+- Update `tendermint` dependencies to `v0.28.0`, which contain an important security fix.
+([#294](https://github.com/cosmos/ibc-rs/issues/294))
+
+### BUG FIXES
+
+- Set counterparty connection ID to None in `conn_open_init` ([#174](https://github.com/cosmos/ibc-rs/issues/174))
+- Verify the message's counterparty connection ID in `conn_open_ack`
+  instead of the store's ([#274](https://github.com/cosmos/ibc-rs/issues/274))
+
+### IMPROVEMENTS
```
Comment (hu55a1n1) [inline CHANGELOG.md:40]: Can we move entries in `.changelog/enhancement` into `.changelog/improvements`?

**Review by hu55a1n1 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (plafer) [reply CHANGELOG.md:47]: I think it's a minor release enough to just let it be? WDYT?

File: CHANGELOG.md (hunk 2 above)
Comment (plafer) [reply CHANGELOG.md:40]: [bd29d96](https://github.com/cosmos/ibc-rs/pull/300/commits/bd29d962ada53e974f954d47d37f5a20f8fcff10)

File: CHANGELOG.md (hunk 1 above)
Comment (plafer) [reply CHANGELOG.md:47]: Or just copy/paste the changelog here?

**Review by hu55a1n1 — APPROVED** [review]

**Pull request merged**